src/main/java/github/scarsz/discordsrv/objects/managers/link/JdbcAccountLinkManager.java

```diff
@@ -125,7 +132,7 @@ public JdbcAccountLinkManager() throws SQLException {
         database = connection.getCatalog();
         String tablePrefix = DiscordSRV.config().getString("Experiment_JdbcTablePrefix");
         if (StringUtils.isBlank(tablePrefix)) tablePrefix = ""; else tablePrefix += "_";
-        accountsTable = "`" + database + "`." + tablePrefix + "accounts";
+        accountsTable = "`" + tablePrefix + "accounts";
```

Comment:
Hanging `

Reply:
done

Reply:
This hasn't been fixed yet

Reply:
huh ? for me it seems fixed
![image](https://user-images.githubusercontent.com/34487011/107145259-ac089880-6940-11eb-8255-00addf181cf6.png)

Reply:
Yes, that is not fixed. There is only one backtick in that image. You can't just only have one.